webui/react/src/utils/browser.ts

```diff
@@ -0,0 +1,7 @@
+export const updateFavicon = (iconPath: string): void => {
+  const linkEl: HTMLLinkElement | null = document.querySelector("link[rel*='shortcut icon']");
+  if (!linkEl) return;
+  linkEl.type = 'image/x-icon';
```

Comment:
the `type` can be either `image/x-icon` (for .ico) and `image/png`.
this should either come through as a parameter in the `updateFavicon` function, or better yet, detect the suffix of the `iconPath` and auto determine the type.